resources/south-america/brazil/mapeadoresIVIDES.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "OSM-group",
+  "type": "group",
+  "locationSet": {
+  
+    "include": ["076"],
+    "include": ["005"],
+    "include": ["019"],
+    "include": ["rio_de_janeiro.geojson"]
+  
+  },
+  "languageCodes": ["pt"],
+  "order": 4,
+  "strings": {
+    "community": "Mapeadores IVIDES",
+    "comunityID": "mapeadoresIVIDES",
+    "description": "All are welcome! To see details, please, visit {signupUrl}",
+    "signupUrl": "https://tasks.hotosm.org/teams/392/membership/",
+    "url": "https://tasks.hotosm.org/manage/organisations/246/"
```

Comment:
this gives login wall if you have no third-party account

Reply:
I fixed the URL, thank you.